docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
       # Django settings overrides:
       - ACCOUNT_DEFAULT_HTTP_PROTOCOL=http
       - ALLOWED_HOSTS=*
+      - ATTACHMENT_HOST=localhost:8000
```

Comment:
The same issue appears in the master branch, so this change doesn't appear to be the cause. I can't recall if I've tried uploads in the docker dev environment. Bug needed.